test/com/facebook/buck/android/testdata/android_project/kotlin/com/sample/lib/BUCK.fixture

```diff
@@ -0,0 +1,35 @@
+android_library(
+  name = 'lib',
+  srcs = glob(['*.kt']),
+  language = 'KOTLIN',
```

Comment:
@zbsz So specifically, no, there should not be any backwards-compatibility issues. I believe we would pass those other types of files to `javac` and the compiler should reject them and fail.